src/common/Configuration/Config.cpp

```diff
@@ -229,8 +239,8 @@ T ConfigMgr::GetValueDefault(std::string const& name, T const& def, bool showLog
     {
         if (showLogs)
```

Comment:
This one must be removed, we DO NOT want to be forced having all the confs of the `.dist` file inside our `.conf`